gemma-teixido/pre-curso/tema-1/calculator-pro.js

```diff
@@ -0,0 +1,114 @@
+let resultArr = [];
+
+function message() {
+  let messageNewNumber = prompt("New numbers? y/n");
+  switch (messageNewNumber) {
+    case "n":
+      let stop = "Bye!";
+      console.log(stop);
+      return stop;
+
+    case "y":
+      let numbers = prompt("Introduce numbers separated by commas");
+      let numberList = numbers.split(",");
+      for (let i = 0; i < numberList.length; i++) {
+        if (isNaN(+numberList[i])) {
+          console.log("Wrong character, try again with a number");
+          return null;
+        }
+      }
+      return numberList;
+  }
+}
+
+function calculatorPro() {
+  let arrayArg;
+
+  do {
+    arrayArg = message();
+  } while (!arrayArg || arrayArg.length === 0);
+
+  if (arrayArg === "Bye!") {
```

Comment:
It is better to declare full word variables instead of shorted. For example, arrayArgument is better than arrayArg, helps at reading code.

There are some few exceptions, for example DB to refer to a Data Base still correct.